src/test/scala/com/microsoft/hyperspace/index/rules/HyperspaceRuleSuite.scala

```diff
@@ -27,17 +28,18 @@ import com.microsoft.hyperspace.HyperspaceException
 import com.microsoft.hyperspace.actions.Constants
 import com.microsoft.hyperspace.index._
 import com.microsoft.hyperspace.index.Hdfs.Properties
+import com.microsoft.hyperspace.util.PathUtils
 
 trait HyperspaceRuleSuite extends HyperspaceSuite {
-  private val filenames = Seq("f1.parquet", "f2.parquet")
```

Comment:
You can change this to `defaultFilenames` and use it as a default parameter.

Reply:
thanks @imback82